.orchestra/support/mass-testing/README.md

```diff
@@ -0,0 +1,46 @@
+# Mass Testing tools
+
+## What's this?
+
+This directory contains tools related to handling binaries used in mass testing
+
+### `extract_binaries`
+
+This tool is used to extract binaries from various sources and upload them to S3.
+The two main parameters that control the behavior:
+* `--extractor`: the script that will take care of extracting files from the
+  source (e.g. unpacking files from an iso image). The extractor can be omitted
+  if the input is a directory.
+* `--filter`: a filtering mechanism to only consider some files. Omitting it
+  will result in all files being uploaded to S3.
+
+#### Currently implemented `extractor`s
+
+* `android_sdat`: extract android zip files from Lollipop (5.0) onwards.
+  Requires [`sdat2img.py`](https://github.com/xpirt/sdat2img) and `p7zip` to
+  be present on `PATH`
+* `android_treble`: extract android zip files which use Project Treble format.
+  Requires [`android-ota-extractor`](https://github.com/tobyxdd/android-ota-payload-extractor)
+  and `p7zip` to be present on `PATH`
+* `linux_live`: extract files from most linux live isos, including squashfs images and snaps.
+  Requires `p7zip`
+* `windows_iso`: extract files from a windows installation iso. Requires
+  [`wimlib`](https://wimlib.net/) and `p7zip` to be installed.
```

Comment:
I think you said you've run these tools yourself, did you?
Can you provide a script to run them? It doesn't have to be perfect, but an example script is worth a 1000 words of explanation.

I'm missing an end-to-end explanation on what I need to do if I want to add, say a new Linux distro.
What commands should I run?
Where/how should I push things?

Reply:
> I think you said you've run these tools yourself, did you?

Yes

> Can you provide a script to run them?
> What commands should I run?
> Where/how should I push things?

I'll add a simple example script to the `README` detailing how the original 5 sources have been imported